src/pages/TCY/components/Stake/StakeInput.tsx

```diff
@@ -157,12 +159,14 @@ export const StakeInput: React.FC<TCYRouteProps & { activeAccountNumber: number
     bnOrZero(amountCryptoPrecision).isZero() ||
     isEstimatedFeesDataError ||
     !amountCryptoPrecision ||
-    !fiatAmount
+    !fiatAmount ||
+    isChainHalted
 
   const confirmCopy = useMemo(() => {
+    if (isChainHalted) return translate('common.poolHalted')
```

Comment:
nitpick - do we want to discern the difference between pool halted and chain halted copy?

Reply:
I think we absolutely do! fbfbbdb214
Probably deserves another pass across the board for current pool halted/chain halted references to ensure we're handling both (e.g here we're not handling inactive pool, and the opposite in other domains, we're only handling pool halted)